src/components/Posts/ExpandedPostReplies.tsx

```diff
@@ -25,13 +26,17 @@ const ExpandedPostReplies = ({ post }: ExpandedPostRepliesProps) => {
             },
           }
         );
-        dispatch(setPosts(resultReplies.data as Post[]));
+        const modified = resultReplies.data.map((entry: any) => ({
```

Comment:
Instead of using `any` as the type for `entry` we should use `Post`. 

Using `any` feels like it takes away from using Typescript and generally isn't really used

Reply:
```suggestion
        const modified = resultReplies.data.map((entry: Post) => ({
```